lib/bootloader_setup.pm

```diff
@@ -337,6 +338,13 @@ sub specific_bootmenu_params {
         }
     }
 
+    # For leap 42.3 we don't have addon_products screen
+    if (!addon_products_is_applicable()) {
+        my $addon_url = get_var("ADDONURL");
+        $addon_url =~ s/\+/,/g;
```

Comment:
In addon_products we have:
foreach my $url (split(/\+/, get_var("ADDONURL"))) { ... }
So, seems that it's possible to pass multiple urls separated with +. Probably we should store this separator in common place not to forget to change it in both places, if we decide to keep this. I've checked testsuites which use ADDONURL, but could not find a single one which uses multiple urls.